modules/imgproc/test/test_thresh.cpp

```diff
@@ -478,6 +478,221 @@ static void test_threshold( const Mat& _src, Mat& _dst,
     }
 }
 
+//test for adaptive Threshold
+static void test_adaptiveThereshold( const Mat& _src, Mat& _dst, double maxValue,
```

Comment:
I don't see any invocation of `cv::adaptiveThreshold` here.

Reply:
Sir, you only told to start writing test function in `test_thresh` and then open a PR.

Reply:
Plus, @dkurt  you to told me told `cv::threshold` as reference. Neither `test_threshold` has an invocation of `cv::threshold`

Reply:
@dkurt . Sir, I don't understand where the values are being verified in the file `test_thresh.cpp`. As there is no code where the expected and actual values from the function are compared in the file. From the code it seems that we only initialized and grabbed the values but never actually compare them. Is there something that is going on internally or am I missing something.